daemon/daemon_unix.go

```diff
@@ -1184,3 +1194,30 @@ func setupOOMScoreAdj(score int) error {
 	f.Close()
 	return err
 }
+
+func (daemon *Daemon) initCgroupsPath(path string) error {
+	if path == "/" || path == "." {
+		return nil
+	}
+
+	daemon.initCgroupsPath(filepath.Dir(path))
+
+	root := "/sys/fs/cgroup/cpu"
+	path = filepath.Join(root, path)
+	sysinfo := sysinfo.New(false)
+	if err := os.MkdirAll(path, 0755); err != nil && !os.IsExist(err) {
+		return err
+	}
+	if sysinfo.CPURealTimePeriod && daemon.configStore.CPURealTimePeriod != 0 {
```

Comment:
Then, since it's a failure of `runC` it should be fixed in `runC`.

Since we are now delegating all this cgroup handling to `runC` it doesn't make sense to bring some of it back into `docker`.